components/content/inspira/ui/text-highlight/TextHighlight.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <span
+    :class="cn('relative inline-block pb-1 px-2 rounded-lg', props.class)"
+    >{{ text }}</span
+  >
+</template>
+
+<script setup lang="ts">
+import type { HTMLAttributes } from 'vue';
+import { cn } from '@/lib/utils';
+
+interface Props {
+  text: string;
+  class: HTMLAttributes['class'];
+}
+
+const props = defineProps<Props>();
+</script>
+
+<style scoped>
+@keyframes backgroundExpand {
```

Comment:
Can we use snake case `background-expand`, to keep it consistent across the components